specification/dartLangSpec.tex

```diff
@@ -2018,25 +2018,51 @@ \subsection{External Functions}
 \LMHash{}%
 An \IndexCustom{external function}{function!external}
 is a function whose body is provided separately from its declaration.
-An external function may be a top-level function (\ref{librariesAndScripts}), a method (\ref{instanceMethods}, \ref{staticMethods}), a getter (\ref{getters}), a setter (\ref{setters}) or a non-redirecting constructor (\ref{generativeConstructors}, \ref{factories}).
-External functions are introduced via the built-in identifier \EXTERNAL{} (\ref{identifierReference}) followed by the function signature.
+An external function may be
+a top-level function (\ref{librariesAndScripts}),
+a method (\ref{instanceMethods}, \ref{staticMethods}),
+a getter (\ref{getters}),
+a setter (\ref{setters}),
+or a non-redirecting constructor
+(\ref{generativeConstructors}, \ref{factories}).
+External functions are introduced via the built-in identifier \EXTERNAL{}
+(\ref{identifierReference})
+followed by the function signature.
```

Comment:
Maybe mention already that a variable declaration (top-level, static or instance) can be marked external, and that it's equivalent to declaring an external getter and (unless `final`) setter.

Reply:
I think I'll do that separately. It would be somewhat bigger, because it involves grammar changes as well, and changes to section 'Variables'.